src/style.css

```
@@ -0,0 +1,20 @@
+html {
```

```suggestion
body {
```

Better to apply font-family in 'body' selector